src/nginx/status.h

```diff
@@ -41,6 +41,7 @@ namespace nginx {
 // The maximum number of esp objects.
 const int kMaxEspNum = 10;
 const int kMaxServiceNameSize = 256;
+const int kMaxServiceRolloutsInfoSize = 256;
```

Comment:
As Wayne mentioned, we can define the maximum number of config_id in a rollout to 10. 

According to my test, if the config_id is generated by the server, 256 seems to be good enough.
But config_id can be assigned by user and maximum length of it is not defined in the service.proto, we might not be able to set long enough.

I increased this to 4096.